index.js

```diff
@@ -0,0 +1,51 @@
+ let date = new Date();
+ let n = date.toDateString();
+ let time = date.toLocaleTimeString();
+ document.getElementById('timeandDate').innerHTML = n + ' ,' + time;
+ document.getElementById('timeandDate').style.color = "white";
+ document.getElementById('timeandDate').style.marginTop = "200px";
+ document.getElementById('timeandDate').style.marginLeft = "250px";
+ document.getElementById('timeandDate').style.fontSize = "25px";
+
+ let temperatureCmp = document.getElementById("weatherTemperature");
+ let nameCmp = document.getElementById("name");
+ let weatherMaxTempCmp = document.getElementById("weatherMaxTemp");
+ let weatherCloudyCmp = document.getElementById("weatherCloudy");
+ let weatherWindCmp = document.getElementById("weatherWind");
+ let weatherSearchBar = document.getElementById("searchBar");
+ let searchBtnCmp = document.getElementById("searchBtn");
+ let weatherIconCmp = document.getElementById("weatherIcon");
+
+ searchBtnCmp.addEventListener("click", function() {
+     fetch('https://api.openweathermap.org/data/2.5/weather?q=' + weatherSearchBar.value + '&units=metric&appid=08a09c29086a3f06cd37337b12b1711f')
+         .then(response => response.json())
+         .then(data => {
+             let currentTemperature = data.main.temp;
+             let maxTempValue = data.main.temp_max;
```

Comment:
you don't need these variables. Because you're using them only once. So you can do something like that:
```
let newMaxTempValue = document.createElement("span");
newMaxTempValue.classList.add("tempMarginAdjustment");
newMaxTempValue.innerHTML = data.main.temp_max;
```

Reply:
Do you mean, instead of maximum temperature , i can include something else like humidity,pressure etc?? As current temperature here refers to main temperature.